.cirrus.yml

```diff
@@ -42,7 +42,7 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
   depends_built_cache:
     folder: "depends/built"
   ci_script:
-    - ./ci/test_run_all.sh
+    - if [ "${CIRRUS_BRANCH}" != "${CIRRUS_DEFAULT_BRANCH}" ] && [ "${CIRRUS_BASE_BRANCH}" != "${CIRRUS_DEFAULT_BRANCH}" ]; then export RUN_FUZZ_TESTS="false"; fi && source ./ci/test_run_all.sh
```

Comment:
I meant that the whole task can be skipped. Compilation will be checked by the other tasks already and there is nothing else done, no?

Reply:
Right. And this approach has one line diff :)

Done in the recent push.